Update documentation of Java Tracker according to recent version (#769)
matomo-org/developer-documentation: docs/5.x/tracking-java.md

---
category: API Reference
---

# Java Tracker

The Matomo Java Tracker is a comprehensive Java library designed for sending tracking data to Matomo servers.
It is compatible with Java 1.8 and later versions. Whether you're working with Java web applications, desktop
applications, command line applications, batch applications, microservices, or other Java-based projects,
the Matomo Java Tracker has you covered.

It differs from the JavaScript Tracker API due to the unique characteristics of Java applications. This library offers
robust support for both synchronous and asynchronous tracking. It facilitates the tracking of custom dimensions, custom
variables, custom events, content, ecommerce activities, goals, site searches, campaign efforts, and more.

For detailed API documentation, refer to the [JavaDoc](https://javadoc.io/doc/org.piwik.java.tracking/matomo-java-tracker-core/latest/index.html).
This documentation provides a swift and informative overview of the essential classes and methods within the library.

If you encounter any challenges or have inquiries, feel free to [create an issue](https://github.com/matomo-org/matomo-java-tracker/issues/new) in

> the [Matomo Java Tracker repository](https://github.com/matomo-org/matomo-java-tracker). Our team is dedicated to
> assisting you promptly and ensuring a smooth experience with the Matomo Java Tracker.

Reviewer: ```suggestion
the [Matomo Java Tracker repository](https://github.com/matomo-org/matomo-java-tracker). Our team will be happy to help.
```
